packages/react/src/create-strict-api/__tests__/__fixtures__/strict-api.ts

```diff
@@ -10,6 +10,15 @@ interface CSSPropertiesSchema {
   bkgrnd: 'red' | 'green';
 }
 
-const { css, XCSSProp, cssMap, cx } = createStrictAPI<CSSPropertiesSchema>();
+type MediaQuery =
+  | '(min-width: 30rem)'
+  | '(min-width: 48rem)'
+  | '(min-width: 64rem)'
+  | '(min-width: 90rem)'
+  | '(min-width: 110rem)'
+  | '(prefers-color-scheme: dark)'
+  | '(prefers-color-scheme: light)';
+
+const { css, XCSSProp, cssMap, cx } = createStrictAPI<CSSPropertiesSchema, { media: MediaQuery }>();
```

Comment:
Ah, I suppose we could restrict our pseudos like this too…I guess that's kinda the issue; we're restricting keys via a passed-around generic…while pseudos are never _extended._

Reply:
Yeah. I suppose one argument for restricting pseudos would be the unsafe "not my ownership" ones.